### DISCUSSION TITLE
Problems with loading custom fonts

### DISCUSSION THREAD

**Discussion by DariusPlaczek** [vercel/next.js#33924] [Help]
Good evening, can someone please tell me how to load custom fonts from the /public/fonts/CustomFonts.woff fonts. Previously you could just go to global.css

   @font face {
   font-family: 'Custom Fonts';
   src: url('./public/fonts/CustomFonts.woff2') format('woff2'),
       url('./public/fonts/CustomFonts.woff') format('woff');
   font-weight: 500;
   font-style: normal;
   font-display: swap;
}

load the fonts. With NextJs Ver. 12 is no longer possible. The documentation only says how to load fonts from Google, but not your own.

Error: "Failed to load resource: the server responded with a status of 404 (Not Found)"

Many thanks for the help.

**Comment by DariusPlaczek** [comment]
Ok I have found the error. I did not see that the files are now in the style folder.

**Reply by rahXephonz** [reply]
Don't forget to give optimizeFont Into your next config file i have some issue the font didnt loaded when i deploying into server whichis vercel
